src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberContainerTypeSymbol.vb

```diff
@@ -1391,7 +1405,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         ''' </summary>
         Public ReadOnly Property SyntaxReferences As ImmutableArray(Of SyntaxReference)
             Get
-                Return _declaration.SyntaxReferences
+                Dim result = _declaration.SyntaxReferences
+                If result.Length >= 2 Then
+                    result = result.OrderBy(Function(ByVal r As SyntaxReference) LocationKey(r.GetLocation())).ToImmutableArray()
+                End If
+                Return result
             End Get
         End Property
```

Comment:
Se https://github.com/dotnet/roslyn/blob/master/src/Compilers/VisualBasic/Portable/Symbols/Source/SourceNamedTypeSymbol.vb#L1308 for one example.